config/webpack.stage.js

```diff
@@ -37,25 +38,33 @@ module.exports = merge(
       })
     ],
     devServer: {
+      ...(OPEN_PATH && { open: [OPEN_PATH] }),
       host: HOST,
       port: PORT,
       compress: true,
       historyApiFallback: true,
       hot: true,
       devMiddleware: {
-        stats: 'errors-only',
+        stats: 'errors-warnings',
         writeToDisk: true
       },
       client: {
         overlay: false,
-        progress: false
+        progress: true
       },
       static: {
         directory: DIST_DIR
       },
       watchFiles: {
         paths: ['src/**/*', 'public/**/*']
-      }
+      },
+      proxy: [
+        {
+          context: ['/api'],
+          target: 'https://0.0.0.0:9443',
```

Comment:
Should this be `127.0.0.1`?

Reply:
same response as above, the actual location of the update happened in the dotenv file `.env` with an update to `HOST=127.0.0.1`

Reply:
Same as above, the dotenv file update appears to have been enough